### PR TITLE
200 ios app docs

### PR DESCRIPTION
**Pull request by dome2048**
Here are my proposed changes to this page, in response to issue #200 . 

Signed-off-by: Tom Nerison <captomner@gmail.com> (github: dome2048)

### CODE REVIEW

File: addons/uis/apps/ios.md [hunk 1]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+* Receive push notifications on your iOS devices via myopenHAB
+* Select a default sitemap
+* Natively supports both iPhone and iPad
+
+**Screenshots:**
+
+<div class="row">
+  <div class="col s12 m5"><img src="images/ios01.png" alt="Demo Main Menu"></div>
+  <div class="col s12 m5"><img src="images/ios02.png" alt="Demo Widget Overview"></div>
+  <div class="col s12 m5"><img src="images/ios03.png" alt="Settings"></div>
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: "m5" means, that the image will take 5/12 of the page width on a medium sized display (a normal PC display). With two images that was okay but you'd need to work with "m4" here

File: addons/uis/apps/ios.md [hunk 2]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
 
 We provide a native iOS app for openHAB, compatible with iPhone and iPad.
 It uses the REST API of openHAB to render sitemaps of your openHAB installation.
-It also supports [myopenhab.org](http://www.myopenhab.org) including push notifications.
+
+**Features:**
+
+* View and control openHAB sitemaps, similar to the BasicUI
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: I'd remove the second part. the iOS app is one of the equal options to view a sitemap.

File: addons/uis/apps/ios.md [hunk 3]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+* Connect to local openHAB instances on the local network
+* Connect to your openHAB instance remotely using myopenHAB
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: Please modify this point or add another making clear that every accessible remote openHAB instance can be used.
Also (the Android offers this option) the app first tries the local, then the external address - that should be mentioned as a feature

File: addons/uis/apps/ios.md [hunk 4]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+* Receive push notifications on your iOS devices via myopenHAB
+* Select a default sitemap
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: Is that a feature? Feels like something not worth mentioning!?

File: addons/uis/apps/ios.md [hunk 5]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+
+**Screenshots:**
+
+<div class="row">
+  <div class="col s12 m5"><img src="images/ios01.png" alt="Demo Main Menu"></div>
+  <div class="col s12 m5"><img src="images/ios02.png" alt="Demo Widget Overview"></div>
+  <div class="col s12 m5"><img src="images/ios03.png" alt="Settings"></div>
+</div>
+
+**Installation and setup:**
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: Please capitalize to read "Installation and Setup"

File: addons/uis/apps/ios.md [hunk 6]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+To connect your iOS app to your openHAB server, navigate to the settings page, then
+disable the "Demo mode" checkbox. In the local URL field, enter either your server's
+IP address (It should look something like 'http://192.168.1.125:8080/') or, if available,
+your server's name (For openHABian, try 'http://openhabianpi.local:8080/').
+
+To access your openHAB server remotely, you'll need to configure myopenHAB. First, create
+an account at [myopenhab](http://www.myopenhab.org). Once you have your username and password, enter them in the appropriate 
+fields. Under the Remote URL field, enter 'https://myopenhab.org/'
+
+**Help and resources:**
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: Please capitalize to read "Help and Resources"

File: addons/uis/apps/ios.md [hunk 7]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
 
 We provide a native iOS app for openHAB, compatible with iPhone and iPad.
 It uses the REST API of openHAB to render sitemaps of your openHAB installation.
-It also supports [myopenhab.org](http://www.myopenhab.org) including push notifications.
+
+**Features:**
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: This and all other headline should be actual headlines...

File: addons/uis/apps/ios.md [hunk 8]
```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
...
+more details.
+
+To connect your iOS app to your openHAB server, navigate to the settings page, then
+disable the "Demo mode" checkbox. In the local URL field, enter either your server's
+IP address (It should look something like 'http://192.168.1.125:8080/') or, if available,
+your server's name (For openHABian, try 'http://openhabianpi.local:8080/').
+
+To access your openHAB server remotely, you'll need to configure myopenHAB. First, create
+an account at [myopenhab](http://www.myopenhab.org). Once you have your username and password, enter them in the appropriate 
+fields. Under the Remote URL field, enter 'https://myopenhab.org/'
```
Comment (ThomDietrich) [inline addons/uis/apps/ios.md]: I'd remove that part. myopenHAB is just one option and if a user uses it, he/she will know the URL to enter as described above

**Comment by ThomDietrich** [comment]
Hey @dome2048, thanks for the corrections. I've added a few myself, could you please check them for me?
From my point of view we are ready to merge. What do you think?

File: addons/uis/apps/ios.md [hunk 9]
```diff
@@ -5,46 +5,47 @@ title: iOS openHAB App
 
 {% include base.html %}
 
-## iOS openHAB App
```
Comment (dome2048) [inline addons/uis/apps/ios.md]: I wasn't sure which convention to use for headings - I used ## for the main header and ### for the sub headings to be in line with what I saw on the BasicUI page, but in looking around there's some variation - I imagine you know best though! 😀

**Comment by dome2048** [comment]
I have no issues with your edits - looks great! Thanks for helping me learn how to contribute. I'm happy to be able to do something to support openHAB.

File: addons/uis/apps/ios.md (hunk 9 above)
Comment (ThomDietrich) [reply addons/uis/apps/ios.md]: In general articles should start with H1. The main articles should do so. I'll look into Basic UI.

**Comment by ThomDietrich** [comment]
Sure thing ;) This is a great addition. Thanks and looking forward to your next contribution ;)
In fact, now that we have this great app article, how about mirroring the information to android? The main content should be identical. Wdyt?

**Comment by dome2048** [comment]
Sure, I'd be happy to do that. It might be a few days before I get to it but I'll send you a new PR when I'm done.

**Comment by ThomDietrich** [comment]
New PR it is. Okay then, here we go :)

**Pull request merged**